flink-python/pyflink/fn_execution/timerservice_impl.py

```diff
@@ -59,6 +69,7 @@ def __init__(self, keyed_state_backend: RemoteKeyedStateBackend):
         self._keyed_state_backend = keyed_state_backend
         self._current_watermark = None
         self.timers = []  # type: List[Tuple[TimerOperandType, InternalTimer]]
+        self._exist_timers = set()  # type: Set[Tuple[TimerOperandType, InternalTimer]]
```

Comment:
What about change the type of timers to OrderedDict, then we can avoid to introduce _exist_timers.